i18n/locales/source/course_content/2019/dance-extras-2019.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
I'd expect this file to be added under `Hour of Code`, not under the `2019` directory; do we know why it's here? Is it supposed to be?

Reply:
Good question!  I'm not sure.  I will check with the starlabs team.